CatchIoScriptImpl/PlayerCharacter/Inventory.cs

```diff
@@ -87,13 +85,39 @@ public bool IsEmpty()
             return _storeItemDictionary.Count == 0;
         }
 
+        public Item[] ToArray()
```

Comment:
This method name is so ambiguous. 
I recommend this method name change InventoryItemToArray or something.